components/scripts/lib/config.sh

```diff
@@ -62,6 +62,10 @@ process_arguments() {
   if [ -n "${_arg_interactive+x}" ]; then
     interactive_mode="${_arg_interactive}"
   fi
+
+  if [ -n "${_arg_offline+x}" ]; then
```

Comment:
I would move this if-block above the interactive if-block.

Reply:
Addressed in c9110cc807f3057942e957a0cd8f3a9066006701.